e2e/pages/selectHearing/selectHearing.js

```diff
@@ -33,7 +37,7 @@ module.exports = {
   },
 
   enterHearingType() {
```

Comment:
Please rename. First it is not data entering. Second method should either say what value is being selected or should accept argument indicating what is being selected.